libs/email/components/content.tsx

```
@@ -2,33 +2,125 @@
+          <td className="px-10 pb-10">
             <Card title={title}>
               <table className="w-full">
                 <tbody>
+                  <tr>
+                    <td className="text-center w-full">
+                      <pre className="text-sm text-[#e6e6e6] font-mono text-left text-wrap">
+                        You're registered for cuHacking 2025 at Carleton University! 🔥🔥🔥
+                        <br></br>
+                        We're excited to have you join us for a weekend of hacking, learning, and fun!!!
+                      </pre>
+                      <a href="https://portal.cuhacking.ca" target="_blank" className="block w-full">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)